libs/components/avatar/src/lib/modules/avatar/avatar.component.ts

```diff
@@ -55,7 +55,7 @@ export class SkyAvatarComponent {
   }
 
   /**
-   * Specifies an image to display to identify a record. This property is
+   * An image to display to identify a record. This property is
```

Comment:
```suggestion
   * The image to display to identify a record. This property is
```
I could be offbase here, but starting with `An` feels off when looking at all the others that start with `The`. I'm pretty sure that we should go with the specific `the` instead of more generic `an` now that we are losing the verb at the start of these descriptions.